### DISCUSSION TITLE
includeSubdomains in Strict-Transport-Security

### DISCUSSION THREAD

**Discussion by NKazantsev** [BePsvPT/secure-headers#64] [Ideas]
Hi,
what about **includeSubdomains** option in  Strict-Transport-Security ?
Can you add it to secure-headers ?
DESCRIPTION
The application is setting the Strict-Transport-Security header but with an insecure value, specifically without the includeSubdomains option. The includeSubdomains option will extend the benefit of the header to the subdomains, preventing situations where the attacker registers (or takes over) a subdomain and leverages that read session cookies from the parent domain.

**Comment by incoming-th** [comment]
I believe this is already from this PR: https://github.com/bepsvpt/secure-headers/pull/15
